tasks/maslov/3_coins.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+def coinChange(coinNeeded):
+   coinValues = [1,3,7,13,29,50]
+   # creating zeroed array in list
+   minCoins = [[0 for j in range(coinNeeded + 1)]
+               for i in range(len(coinValues))]
+   # transforming first element of list array to range
+   minCoins[0] = range(coinNeeded + 1)
+
+   # trying coins valuse
+   for i in range(1,len(coinValues)):
```

Comment:
usually, they use `xrange` in Python because it avoids additional computation and space usage for the range list. It is not an issue here because the number of coins is small, but, I believe, it's a good default choice not to forget it by chance where it will matter